frontend/src/component/FeedItem/FeedItemFooter.jsx

```diff
@@ -8,7 +8,8 @@ import {
   ROLE_LIST,
   TRANSLATION_STATE,
   TranslateButton,
-  EmojiReactions
+  EmojiReactions,
+  IconButton
```

Comment:
It's normal that IconButton was added here but not in the rest of the file?

Reply:
It's the component I used for the 'Show discussion' button

Reply:
But it's called only on `Comment.jsx`, no?

Reply:
my bad, you are right :) thanks